gdcdatamodel/models/__init__.py

```diff
@@ -17,6 +17,8 @@
 import os
 import sys
 
+import six
```

Comment:
We are still supporting python 2?

Reply:
Sadly we are, [biowcs](https://github.com/NCI-GDC/biowcs) is still py2 and it is a mojor player for GPAS